### ISSUE TITLE
Resize SD card

### ISSUE THREAD

**Issue by Rick88Stang** [ludiazv/octocitrico#27]
Does this automatically resize the SD card on first run to use the entire SD card? I want to run a 32GB card for octoprint.

**Comment by catallo** [comment]
Yes it does. Although the newer versions don't seem to work correctly because of a syntax change in HAProxy. So I'd recommend to install an older version like https://github.com/ludiazv/octocitrico/releases/tag/v20.11-1

**Comment by Rick88Stang** [comment]
Ty I will give it a shot next week as somehow I messed up my install via online guides and now I can't install addons...

**Comment by ludiazv** [comment]
The image resizes the filesystem on the first boot. The problem with HA is fixed and will be available in the next releas.
